pgjdbc/src/test/java/org/postgresql/test/jdbc2/DriverTest.java

```diff
@@ -225,7 +201,7 @@ public void testRegistration() throws Exception {
 
     drivers = Collections.list(DriverManager.getDrivers());
     for (java.sql.Driver driver : drivers) {
-      if (driver instanceof software.aws.rds.jdbc.postgresql.Driver) {
+      if (driver.getClass().equals(org.postgresql.Driver.class)) {
```

Comment:
Had to change this since instanceof would result in true if the aws driver was registered (since aws register is a subclass of org.postgresql.Driver)